src/babashka/fs.cljc

```diff
@@ -815,3 +815,31 @@
        ~@body
        (finally
          (delete-tree ~binding-name)))))
+
+(let [homedir (path (System/getProperty "user.home"))
```

Comment:
We can't use the homedir at load time since this won't work well with GraalVM since this will contain the homedir of the compilation environment. Instead, we have to just make homedir and usersdir a function or a delay. Since these values are private, a delay will do. 
So write:
```
(let [homedir (delay (path ...))] ..)
``

Reply:
I got the idea. I will make the necessary adjustement.
And I think this was an optimization. We can always fetch this information directly in the body of the function.
I'll try the delay and fallback to directly reading the property if it does not work.